utils/rowgroup/rowgroup.h

```diff
@@ -1165,12 +1152,7 @@ inline void Row::setLongDoubleField(const long double& val, uint32_t colIndex)
 
 inline void Row::setInt128Field(const int128_t& val, uint32_t colIndex)
 {
-    *((int128_t*)&data[offsets[colIndex]]) = val;
-}
-
-inline void Row::setUint128Field(const uint128_t& val, uint32_t colIndex)
-{
-    *((uint128_t*)&data[offsets[colIndex]]) = val;
+    setBinaryField<int128_t>(&(const_cast<int128_t&>(val)), colIndex);
```

Comment:
I think the first parameter of `setBinaryField` should be a `const T*`.

Reply:
Agree.